app/models/system_console.rb

```diff
@@ -56,7 +56,7 @@ def self.launch_proxy(remote_address, remote_port)
     command = '/usr/bin/socat', "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"
     _log.info("Running socat proxy command: #{command.join(' ')}")
 
-    pid = spawn(*command)
+    pid = spawn(AwesomeSpawn::CommandLineBuilder.new.build(command))
```

Comment:
Don't think this works.

```ruby
02:46:54 ~/projects/external/openapi-generator/samples/client/petstore/crystal (master #) (2.6.5) ✓ irb
[1] pry(main)> require "awesome_spawn"
=> true
[2] pry(main)> command = "echo", "foo", "bar"
=> ["echo", "foo", "bar"]
[3] pry(main)> spawn(AwesomeSpawn::CommandLineBuilder.new.build(command))
=> 15147
[4] pry(main)> sh: [echo,: command not found

```

Reply:
Think this should be `spawn(AwesomeSpawn::CommandLineBuilder.new.build(command[0], command[1..-1]))`

Reply:
Can you also change line 56 to be a proper Array?  This notation works surprisingly, but it's melting my head :)

```ruby
x = 1, 2, 3
x
# => [1, 2, 3]
```

Reply:
So this was added by MartinP a while ago, right? And we're not testing any of it.